### DISCUSSION TITLE
ID für API notwendig?

### DISCUSSION THREAD

**Discussion by pbroska** [Digitaler-Impfnachweis/certification-apis#130] [Q&A] [answered]
Welche ID muss für die API verwendet werden? Alle meine bekannten sind not whitelisted.

**Comment by nicolaslang1** [comment, marked as answer]
Welche API verwenden Sie ? PU, RU oder Public?

**Reply by pbroska** [reply]
Ich teste gerade auf der RU!

**Reply by pbroska** [reply]
Für Public wäre ich auch dankbar, hauptsache ich kann ein Zertifikat erstellen. Der Token ist da!

**Reply by nicolaslang1** [reply]
Ich bin gleich Zuhause dann schicke ich ihnen meine Konfiguration

**Reply by nicolaslang1** [reply]
Für Public gibt es grad einen anderen Thread, da die angegeben Client_ID nicht korrekt ist. Wir warten da leider auch schon lange drauf...

**Reply by nicolaslang1** [reply]
Sorry hatte gelesen dass Sie die Konfiguration für PU wollen nicht für Public, für PU funktioniert es nämlich.

**Reply by pbroska** [reply]
Dankeschön! Welche Parameter verwenden Sie für RU? Auf die PU haben wir aktuell noch keinen Zugang.

**Reply by nicolaslang1** [reply]
Die RU haben wir nicht getestet, da uns keine Konnektoren für die RU zur Verfügung standen, also kann ich Ihnen da leider keine Auskunft geben.

**Reply by pbroska** [reply]
Danke nochmal für Ihre Konfiguration. In der RU kann ich nun ein PDF generieren, Public funktioniert nicht -> wie erwartet.
